<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="6" total="10" passed="2" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-10-24T16:10:53 MSK" name="All Test Suite" finished-at="2022-10-24T16:11:16 MSK" duration-ms="23200">
    <groups>
    </groups>
    <test started-at="2022-10-24T16:10:53 MSK" name="D:/Code/java/Frameworks/src/test/resources" finished-at="2022-10-24T16:11:16 MSK" duration-ms="23200">
      <class name="tests.casesTest.ProblemUserTest">
        <test-method signature="checkIsRedirectToListing()[pri:0, instance:tests.casesTest.ProblemUserTest@7ccfdaef]" started-at="2022-10-24T16:10:59 MSK" name="checkIsRedirectToListing" finished-at="2022-10-24T16:11:02 MSK" duration-ms="2689" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [6] but found [3]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [6] but found [3]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1240)
at org.testng.Assert.assertEquals(Assert.java:1274)
at pages.base.BasePage.addAndRemoveElements(BasePage.java:76)
at pages.OurCases.listing.CasesListingPage.checkAddBtn(CasesListingPage.java:74)
at pages.OurCases.listing.CasesListingPage.doAllActions(CasesListingPage.java:121)
at tests.casesTest.ProblemUserTest.checkIsRedirectToListing(ProblemUserTest.java:17)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkIsRedirectToListing -->
      </class> <!-- tests.casesTest.ProblemUserTest -->
      <class name="tests.casesTest.LockedOutUserTest">
        <test-method signature="checkIsRedirectToListing()[pri:0, instance:tests.casesTest.LockedOutUserTest@49fb0bbd]" started-at="2022-10-24T16:10:58 MSK" name="checkIsRedirectToListing" finished-at="2022-10-24T16:10:59 MSK" duration-ms="1627" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [6] but found [0]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [6] but found [0]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1240)
at org.testng.Assert.assertEquals(Assert.java:1274)
at pages.OurCases.listing.CasesListingPage.checkCountCards(CasesListingPage.java:49)
at pages.OurCases.listing.CasesListingPage.doAllActions(CasesListingPage.java:120)
at tests.casesTest.LockedOutUserTest.checkIsRedirectToListing(LockedOutUserTest.java:17)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkIsRedirectToListing -->
      </class> <!-- tests.casesTest.LockedOutUserTest -->
      <class name="tests.casesTest.StandartUserTest">
        <test-method signature="checkIsRedirectToListing()[pri:0, instance:tests.casesTest.StandartUserTest@dd4aec3]" started-at="2022-10-24T16:10:53 MSK" name="checkIsRedirectToListing" finished-at="2022-10-24T16:10:58 MSK" duration-ms="4828" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkIsRedirectToListing -->
        <test-method is-config="true" signature="clearCookiesAndLocalStorage()[pri:0, instance:tests.casesTest.StandartUserTest@dd4aec3]" started-at="2022-10-24T16:11:16 MSK" name="clearCookiesAndLocalStorage" finished-at="2022-10-24T16:11:16 MSK" duration-ms="32" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clearCookiesAndLocalStorage -->
        <test-method is-config="true" signature="close()[pri:0, instance:tests.casesTest.StandartUserTest@dd4aec3]" started-at="2022-10-24T16:11:16 MSK" name="close" finished-at="2022-10-24T16:11:16 MSK" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- tests.casesTest.StandartUserTest -->
      <class name="tests.casesTest.PerformanceGlitchUserTest">
        <test-method signature="checkIsRedirectToListing()[pri:0, instance:tests.casesTest.PerformanceGlitchUserTest@654e6a90]" started-at="2022-10-24T16:11:02 MSK" name="checkIsRedirectToListing" finished-at="2022-10-24T16:11:16 MSK" duration-ms="13658" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkIsRedirectToListing -->
      </class> <!-- tests.casesTest.PerformanceGlitchUserTest -->
    </test> <!-- D:/Code/java/Frameworks/src/test/resources -->
  </suite> <!-- All Test Suite -->
</testng-results>
